webix.js

```diff
@@ -82,22 +102,59 @@ const formBtns = {
             view: "button",
             value: "Add new",
             css: "webix_primary",
+            click: function() {
+                const formValidation = $$("myForm").validate();
+                if (formValidation) {
+                    const values = $$("myForm").getValues();
+                    $$("myTable").add(values);
+                    webix.message('New film was added');
+                }
+                
+            }
         },
         
         {
             view: "button",
             value: "Clear",
+            click: function() {
+                webix.confirm({
+                    title: "Clear",
+                    id: "myConfirm",
+                    text: "Do you want clear the table?"
+                }).then(
+                    function() {
+                        $$('myForm').clear();
```

Comment:
Validation marks should be cleared as well

Reply:
done